# Information specific to each force sensor.
# Readout is in Newtons
string data
